src/TensorFlowNET.Keras/Utils/generic_utils.cs

```diff
@@ -51,6 +56,85 @@ public static JObject serialize_keras_object(IKerasConfigable instance)
             return serialize_utils.serialize_keras_class_and_config(instance.GetType().Name, config, instance);
         }
 
+        public static Layer deserialize_keras_object(string class_name, JToken config)
```

Comment:
We can dynamically create an Layer object and add some behavior in `IKerasConfig`.